### PR TITLE
downloads: add feather wallet

### PR DESCRIPTION
**Pull request by rating89us**
To be merged when Feather wallet is out of beta.

Changes:
- add feather wallet to light wallets section
- sort wallets in alphabetic order
- fix "GitHub icon" aria-label of remaining wallets
- remove "(beta)" from feather wallet on hardware wallets section

### CODE REVIEW

**Review by selsta — APPROVED** [review]

**Comment by erciccione** [comment]
@rating89us if we are waiting for Feather to go out of beta, please convert this PR to draft.

**Comment by selsta** [comment]
ping @rating89us please put this out of draft mode

File: downloads/index.md [hunk 1]
```diff
@@ -247,6 +247,12 @@ permalink: /downloads/index.html
                             <td>X</td>
                             <td><a class="ext-noicon" href="https://github.com/cake-tech/cake_wallet" aria-label="GitHub icon"><span class="icon-github"></span></a></td>
                         </tr>
+                        <tr>
+                            <td><img class="small" src="/img/feather.png" width="100" height="100" loading="lazy" alt="Feather logo"><a href="https://featherwallet.org/">Feather</a></td>
+                            <td>X</td>
+                            <td><span class="icon-linux"></span><span class="icon-windows"></span><span class="icon-apple"></span></td>
+                            <td><a class="ext-noicon" href="https://git.featherwallet.org/feather/feather" aria-label="GitHub icon"><span class="icon-github"></span></a></td>
```
Comment (erciccione) [inline downloads/index.md:254]: Replace the github icon with the generic git icon:

```suggestion
                            <td><a class="ext-noicon" href="https://git.featherwallet.org/feather/feather" aria-label="GitHub icon"><span class="icon-git"></span></a></td>
```

**Review by erciccione — CHANGES_REQUESTED** [review]
Why adding it in the middle instead of the end of the list? Not a big deal, just curious to know if there is a particular reason

**Comment by erciccione** [comment]
Recovered with #1885 (added my change as well)

**Pull request closed without merging**